org.adempiere.base/src/org/compiere/model/MAttachment.java

```diff
@@ -222,9 +222,10 @@ public MAttachment(MAttachment copy)
 
 	/**
 	 * If the related record is on System and the user is operating on Tenant, the attachment is read-only
+	 * @param isDelete
 	 * @return
 	 */
-	public boolean isReadOnly() {
+	public boolean isReadOnly(boolean isDelete) {

```

Comment:
since isReadOnly() is a public method, should keep the method to be backward compatible.

Reply:
> since isReadOnly() is a public method, should keep the method to be backward compatible.

Thanks @hengsin - fixed